content/running_bookwyrm/install-prod-dockerless.md

```diff
@@ -0,0 +1,147 @@
+---
+Title: Installing in Production
```

Comment:
This title should be something like "Installing Without Docker" to differentiate it from the standard Docker-ful instructions